Javascript and CSS changes

xolf opened the pull request.

Commit:
Added Logo Example

aquilax commented:
Isn't it too big ?

xolf commented:
It's just for preview.

aquilax commented:
Even for preview it's half a MB. Better use this:

![](http://i.imgur.com/Lfm1Vuh.gif)

xolf commented:
Without talking about the design. Do you agree with this logo?

aquilax commented:
Isn't the logo all about design? Still I think you should ask Richmond about this. My concern is only about the share size of the image :signal_strength:

Commit:
Improvemnts with the nav and javascript

Focus is now on the search bar in the nav
Added data attribute for inputs for getting the json file.
This is going to be more improved in the following days.

xolf commented:
I've removed the logo now and made some changes on the nav and welcome page.

Commit:
Added Lato to the font stack

aquilax commented:
LGTM :+1:
